spotify

```diff
@@ -208,9 +207,17 @@ while [ $# -gt 0 ]; do
                     newvol=0;
                     cecho "Spotify volume level is at min.";
                 fi
-            elif [ $2 -gt 0 ]; then
+            elif [[ $2 =~ ^[0-9]+$ ]] && [[ $2 -ge 0 && $2 -le 100 ]]; then
                 newvol=$2;
-                text="to $2";
+                cecho "Setting Spotify volume level to $newvol";
+            else
+                echo "Improper use of \`vol\` command"
```

Comment:
Just a stylistic choice for ease of use. Maybe replace `\`` with `'`. e.g. "Improper use of 'vol' command".